weaviate/datadog_checks/weaviate/check.py

```diff
@@ -0,0 +1,124 @@
+# (C) Datadog, Inc. 2023-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import copy
+import time
+from urllib.parse import urljoin
+
+from datadog_checks.base import AgentCheck, OpenMetricsBaseCheckV2
+from datadog_checks.base.utils.common import round_value
+from datadog_checks.weaviate.config_models import ConfigMixin
+
+from .metrics import METRICS
+
+DEFAULT_METADATA_ENDPOINT = '/v1/meta'
+DEFAULT_NODE_METRICS_ENDPOINT = '/v1/nodes'
+DEFAULT_LIVENESS_ENDPOINT = '/v1/.well-known/live'
+
+# Mapping service checks to HEALTHY = Ok, UNHEALTHY = Warning, UNAVAILABLE = Critical.
+# Defaults to unknown if not of the above (3).
+NODE_STATUS_VALUES = {'HEALTHY': 0, 'UNHEALTHY': 1, 'UNAVAILABLE': 2, 'UNKNOWN': 3}
+
+
+class WeaviateCheck(OpenMetricsBaseCheckV2, ConfigMixin):
+    DEFAULT_METRIC_LIMIT = 0
+    __NAMESPACE__ = 'weaviate'
+
+    def __init__(self, name, init_config, instances=None):
+        super(WeaviateCheck, self).__init__(
+            name,
+            init_config,
+            instances,
+        )
+        self.tags = self.instance.get('tags', [])
```

Comment:
Since you're using `ConfigMixin`s you don't need to get the instance values, they are already set to `self.config.tags` and `self.config.weaviate_api_endpoint`.

Reply:
I tried that before. But I can't seem to get it working. `self.config` for me seems to be None when I try to running it. 

```
AttributeError: 'NoneType' object has no attribute 'tags'
```

Reply:
It might depend on where you are calling `self.config`. The `ConfigMixin` loads the config as a [check initialization](https://datadoghq.dev/integrations-core/meta/config-models/#loading), so it won't be available by the time you call `__init__()`, but it will be ready by `check()`.